libs/FileManagerKit/source/FileManagerKit.cpp

```diff
@@ -20,3 +20,10 @@ auto FileManagerKit::remove(const std::filesystem::path &path) -> bool
 {
 	return std::filesystem::remove(path);
 }
+
+void FileManagerKit::clear(const std::filesystem::path &path)
+{
+	if (std::filesystem::exists(path)) {
```

Comment:
this will suffer from the same issue we had in the other PR, bringing a lot of STL code.

I would prefer we don't add it in this PR, especially without adding a use case in the spike or functional test to monitor the impact.

Reply:
or use the same workaround you used in the other PR.